gmso/tests/test_gro.py

```diff
@@ -46,3 +47,34 @@ def test_write_gro_non_orthogonal(self):
         top = from_parmed(pmd.load_file(get_fn("ethane.gro"), structure=True))
         top.box.angles = u.degree * [90, 90, 120]
         top.save("out.gro")
+
+    @pytest.mark.skipif(not has_mbuild, reason="mBuild not installed.")
+    def test_benzene_gro(self):
+        import mbuild as mb
+        from mbuild.packing import fill_box
+
+        from gmso.external import from_mbuild
+
+        benzene = mb.load(get_fn("benzene.mol2"))
+        benzene.children[0].name = "Benzene"
+        box_of_benzene = fill_box(compound=benzene, n_compounds=5, density=1)
+        top = from_mbuild(box_of_benzene)
+        top.save("benzene.gro")
+
+        reread = Topology.load("benzene.gro")
+        for site, ref_site in zip(reread.sites, top.sites):
+            assert site.molecule.name == ref_site.molecule.name[:3]
+            assert site.molecule.number == ref_site.molecule.number
```

Comment:
Unless I am missing it, I think we need a benzene UA one comparing it to the stored one (ref_site) for comparison.  Or am I missing it?

Reply:
test added